mm2src/coins/utxo/utxo_tests.rs

```diff
@@ -584,3 +584,19 @@ fn get_tx_details_doge() {
     spawn(fut2);
     loop {};
 }
+
+#[test]
+fn get_tx_details_coinbase_transaction() {
```

Comment:
Could you please add link to related Github issue similar to: https://github.com/KomodoPlatform/atomicDEX-API/commit/a7f428afe66eadea769704f7da3b326c1420788f#diff-5059b1a76e237fd84822c2c8f349455aR1976?
It will help to understand context of the test in the future.

Reply:
Done